internal/configs/version1/nginx-plus.tmpl

```diff
@@ -21,6 +21,7 @@ load_module modules/ngx_http_app_protect_module.so;
 {{- if .AppProtectDosLoadModule}}
 load_module modules/ngx_http_app_protect_dos_module.so;
 {{- end}}
+load_module modules/ngx_fips_check_module.so;
```

Comment:
does this adds fips to all the images?

Reply:
This module only prints the line in the logs saying if FIPS is enabled or not. I've added it to all the plus images (it's only a few KBs)

Reply:
That lines up with what @vepatel showed us in the logs from his testing. Thanks for confirming.